contracts/Vault.vy

```diff
@@ -1082,7 +1086,6 @@ def withdraw(
             if strategy == ZERO_ADDRESS:
                 break  # We've exhausted the queue
 
-            vault_balance: uint256 = self.token.balanceOf(self)
```

Comment:
Think you might run into an implicit assumption here that could be causing issues

Reply:
sorry i don't follow, what kind of issues?
`vault_balance` is defined above using `self.totalIdle`